PyRoute/Star.py

```diff
@@ -249,14 +251,13 @@ def parse_line_into_star(line, sector, pop_code, ru_calc):
 
         star.tradeCode.check_world_codes(star)
 
-        if (data[5]):
+        if data[5]:
             imp = int(data[5][1:-1].strip())
             star.calculate_importance()
             if imp != star.importance:
                 star.logger.warning(
-                    '{}-{} Calculated importance {} does not match generated importance {}'.format(star, star.baseCode,
-                                                                                                    star.importance,
-                                                                                                    imp))
+                    '{}-{} Calculated importance {} does not match generated importance {}'.
```

Comment:
Please use the python format strings whenever possible. But only if you end up changing them. You can remove the call to format(...) as well. 
```suggestion
                    f'{star}-{star.baseCode} Calculated importance {star.importance} does not match generated importance {imp}'
```

Reply:
I inherited this one from you, so I didn't substantively touch it.  Maybe do another PR that just straightens all of this string stuff out?